client/landing/stepper/declarative-flow/trial-wooexpress-flow.ts

```diff
@@ -17,6 +20,7 @@ const wooexpress: Flow = {
 		return [
 			{ slug: 'siteCreationStep', component: SiteCreationStep },
 			{ slug: 'processing', component: ProcessingStep },
+			{ slug: 'assignTrialPlan', component: AssignTrialPlanStep },
```

Comment:
Question: what are we waiting for in the processing step?

Reply:
The actual site creation happens in the processing step. Basically the site creation step queues up all the stuff that needs to happen, then it actually gets processed in the processing step.

I initially attempted to make the trial plan assignment happen the same way and have processing happen last but there didn't seem to be a way to make the queued actions happen in a particular order.